src/Controller/RequestOptionsController.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
If this is meant for assertion requests i would rename it to AssertionRequestOptonsController for consistency with the AttestationControllers

Reply:
I kept this controller. I'm quite OK with its name.